README.md

```diff
@@ -91,16 +91,15 @@ The Azure Pipelines Infracost tasks can be used with either Azure Repos (only gi
           - bash: infracost breakdown --path=$(TF_ROOT)/plan.json --format=json --out-file=/tmp/infracost.json
             displayName: Run Infracost
 
-          # See https://github.com/infracost/infracost-azure-devops#infracostcomment for other options
-          - task: InfracostComment@0
+          # Adds a cost estimate comment to a Azure Repos pull request.
```

Comment:
To have the same style as other comments:
```suggestion
          # Add a cost estimate comment to a Azure Repos pull request.
```

Also, only line 90 we probably don't need the `Combine Infracost JSON files` comment, as `infracost comment` can do it automatically.

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/45/commits/51e7994d795d728acc2fcd1bef0007f8f0344df2